infra/cifuzz/fuzz_target.py

```diff
@@ -134,6 +134,11 @@ def _save_crash(self, crash):
     with open(bug_summary_artifact_path, 'w') as handle:
       handle.write(crash.stacktrace)
 
+    # Set permissions of testcase to be the same as summary so that we're sure
+    # it can be read by necessary users.
+    permissions_mode = os.stat(bug_summary_artifact_path)
```

Comment:
This is wrong and causing CI failures :/ 

```
>     os.chmod(target_reproducer_path, permissions_mode)
E     TypeError: an integer is required (got type os.stat_result)
```

os.stat returns a structure:  https://docs.python.org/3/library/os.html#os.stat